packages/installer/upgrade.sh

```diff
@@ -165,7 +165,6 @@ upgrade() {
     # FIXME: Define default /var/tmp as tmpdir_base in default luet config file
     export XDG_RUNTIME_DIR=$temp_upgrade
     export TMPDIR=$temp_upgrade
-    export LUET_PRIVILEGED_EXTRACT=true
```

Comment:
that looks more like a regression, when I've tested this it was not failing to upgrade at all - but it succeeded in the upgrade - the failing aspect was vagrant ssh not working with pubkeys. If now it works without pubkeys, doesn't necessarly mean its bug free in there :)

Reply:
Can you try to check if after an upgrade you can still ssh with the vagrant pubkeys?

Reply:
see here: https://github.com/rancher-sandbox/cOS-toolkit/pull/260#issuecomment-862213314 and https://github.com/rancher-sandbox/cOS-toolkit/pull/260#issuecomment-862216012

My test was simply mimicking CI:

- install
- do upgrade (which succeeded, and I could boot just fine afterwards)
- try to vagrant ssh with a priv/pubkey that was previously there already (setted up via cloud-init)

Reply:
as mentioned on the issue we dont use vagrant keys AFAIK, when you do vagrant ssh it uses the user/pass in the vagrantfile

Maybe when it happened we were not using the ssh user/pass ?

Reply:
Indeed, original issue had vagrant using its own method for the ssh keys while this PR https://github.com/rancher-sandbox/cOS-toolkit/commit/c3d5ba246358b42830e26ec10b627127d82c5ba0#diff-225f3d44285006fdfb0e6a444342ca7f84f2b8f366ae15fa4360265b535be386 changed that to user/pass to avoid spending time in injecting the keys....

Reply:
FYI, this was failing without this patch still, so it should be valid. Doesn't fix #481 thought... :D

Reply:
yes that's why I'm asking to test this with ssh keys :slightly_smiling_face:  the problem was not present by logging with user/pass, but it was there only while logging in via ssh keys...